### DISCUSSION TITLE
How   can I send Duty%  value in slier bar to Fuji PXF   register ?

### DISCUSSION THREAD

**Discussion by Terracotta-19** [artisan-roaster-scope/artisan#879] [Q&A] [answered]
hi MAKOMO 
I  am using Fuji PXF  5AEY2-1WM00 to control my roaster, I would like to control the Fuji PXF in manual mode , control the heater power manually (with Fuji command  pidmode(0)), to hit the  SV  I use PID control of course , PC communications
With Fuji PXF in  Modbus RTU protocol.

Here following is what I did :
1.I have also set two button ,call MV0 to MV100  modbus command : buttons  write(1,2131,0)  .....write(1,2131,100) set  DUTY% value ,and they works .

2.I also would like control DUTY% by the slider  modbus command :write(1,2131,{}) ) , but  is NOT working ,the value data can not send to Fuji PXF registers .
  
 FYI , Here is my *.aset file
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/8644392/artisan-settings.aset.txt)

**Comment by MAKOMO** [comment, marked as answer]
Your slider values range from 0-105. Thus the maximum values assigned to the placeholder {} is 105. Looking at your MV button definition it seems that for MV100 you send `write(1,2131,1000)`. Thus the intended value multiplied by 10. Thus you need to add set the sliders factor to 10 and you should also choose the MODBUS Command action as otherwise the slider will not send out anything.

**Comment by Terracotta-19** [comment]
Thank you very much for your reply, the problem has been solved!
